### PR TITLE
FlippingBinary/issue83

### LINKED ISSUE #83: bug: 'Could not resolve "./headers"'
# Bug

This bug can be reproduced by attempting to build the new SK app in `./demo`. It was introduced by 5a6a64f42349aad0da2d61b11e63189130eaf1dd because it adds a new `headers` module without making `adapt(...)` copy it to the build directory. 

# Proposed solution

Copy all the files in `./files` to the build directory:

```js
builder.copy(files, tmp, {
	replace: {
		SERVER: `${relativePath}/index.js`,
		MANIFEST: './manifest.js',
		DEBUG: debug.toString()
	}
});
```

It would replace these two code blocks:

https://github.com/geoffrich/svelte-adapter-azure-swa/blob/24365cbf315738dbad54093feac97aa119d31a9c/index.js#L63-L69

https://github.com/geoffrich/svelte-adapter-azure-swa/blob/24365cbf315738dbad54093feac97aa119d31a9c/index.js#L78

# Benefits

The provided constants `SERVER`, `MANIFEST`, and `DEBUG` could be used in more than just `entry.js`.

# Downside

This would take up more space in the build directory for files that aren't needed there.

### PR DESCRIPTION
**Pull request by FlippingBinary**
This pull request fixes #83.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -75,8 +75,6 @@ export default function ({
 				})};\n`
 			);
 
-			builder.copy(join(files, 'api'), apiDir);
```
Comment (geoffrich) [inline index.js:78]: I think we still need this line - without it, we no longer copy over `/files/api/function.json` to `api/render`.

File: .github/workflows/azure-static-web-apps-salmon-coast-008650b0f.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+name: Azure Static Web Apps CI/CD
```
Comment (geoffrich) [inline .github/workflows/azure-static-web-apps-salmon-coast-008650b0f.yml:1]: Did you mean to include this file in the PR?

File: index.js (hunk 1 above)
Comment (FlippingBinary) [reply index.js:78]: You're right. I got mixed up about which folder that was copying.

**Review by geoffrich — COMMENTED** [review]
Oops, I think you deleted the wrong file - `release.yml` should not be deleted, `azure-static-web-apps-salmon-coast-008650b0f.yml` should be

**Comment by FlippingBinary** [comment]
Caffeine must not have kicked in when I deleted the wrong file. I think it's right now.

**Review by geoffrich — APPROVED** [review]

**Pull request merged**